src/server.cpp

```diff
@@ -2859,7 +2859,7 @@ void Server::handleChatInterfaceEvent(ChatEvent *evt)
 }
 
 std::wstring Server::handleChat(const std::string &name, const std::wstring &wname,
-	std::wstring wmessage, bool check_shout_priv, RemotePlayer *player)
+	std::wstring &wmessage, bool check_shout_priv, RemotePlayer *player)
```

Comment:
no, you will modify original string as it's not constant

Reply:
Yes, which is why I haven't made it a `const`. That being said, does our code-style even allow non-const references? :thinking:

Reply:
it's not a problem of code style, you will modify the calling passed argument, which is not what we want here.

Reply:
I understand, thanks